### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
## What is the purpose of this change?

Fix misspellings

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [ ] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

#### Notes

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/rclone/commit/f5b15d278498179bf6ce1b6bda97db2d570a8c6a#comments

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/rclone/commit/5e73738725f8331e39d62152fce2649a80ceb94b

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I've tried to be careful to not mess up corrections, but most were picked by Google Sheets and applied automatically, it's quite possible I've accidentally damaged/corrupted things (although the tests past). I'll try to highlight places where I'm less certain about my corrections.

I'm happy to squish/split/reword etc.

### CODE REVIEW

File: MANUAL.html [hunk 1]
```diff
@@ -2092,7 +2092,7 @@ <h2 id="server-options-2">Server options</h2>
 </tr>
 <tr class="even">
 <td style="text-align: left;"></td>
-<td style="text-align: left;">Sort Options: namedirfist,name,size,time (default namedirfirst)</td>
+<td style="text-align: left;">Sort Options: namedirfirst,name,size,time (default namedirfirst)</td>
```
Comment (jsoref) [inline MANUAL.html:2095]: I'm not certain about this

File: MANUAL.html [hunk 2]
```diff
@@ -6565,7 +6565,7 @@ <h3 id="core-command">core/command: Run a rclone terminal command over rc.</h3>
         - Asia Pacific (Mumbai)
         - Needs location constraint ap-south-1.
     - &quot;ap-east-1&quot;
-        - Asia Patific (Hong Kong) Region
+        - Asia Pacific (Hong Kong) Region
```
Comment (jsoref) [inline MANUAL.html:6568]: ?

File: backend/jottacloud/api/types.go [hunk 3]
```diff
@@ -153,9 +153,9 @@ type CustomerInfo struct {
 	AccountType       string      `json:"account_type"`
 	SubscriptionType  string      `json:"subscription_type"`
 	Usage             int64       `json:"usage"`
-	Qouta             int64       `json:"quota"`
+	Quota             int64       `json:"quota"`
 	BusinessUsage     int64       `json:"business_usage"`
-	BusinessQouta     int64       `json:"business_quota"`
+	BusinessQuota     int64       `json:"business_quota"`
```
Comment (jsoref) [inline backend/jottacloud/api/types.go:158]: I'm hoping these aren't public

File: backend/jottacloud/jottacloud.go [hunk 4]
```diff
@@ -107,7 +107,7 @@ func init() {
 				}
 			}
 
-			fmt.Printf("Use legacy authentification?.\nThis is only required for certain whitelabel versions of Jottacloud and not recommended for normal users.\n")
+			fmt.Printf("Use legacy authentication?.\nThis is only required for certain whitelabel versions of Jottacloud and not recommended for normal users.\n")
```
Comment (jsoref) [inline backend/jottacloud/jottacloud.go:110]: This is an output change, but the internet thinks no one will notice

File: backend/onedrive/api/types.go [hunk 5]
```diff
@@ -254,7 +254,7 @@ type MoveItemRequest struct {
 //Always Type:view and Scope:anonymous for public sharing
 type CreateShareLinkRequest struct {
 	Type  string `json:"type"`            //Link type in View, Edit or Embed
-	Scope string `json:"scope,omitempty"` //Optional. Scope in anonymousi, organization
+	Scope string `json:"scope,omitempty"` //Optional. Scope in anonymous, organization
```
Comment (jsoref) [inline backend/onedrive/api/types.go:257]: ?

File: backend/union/union.go [hunk 6]
```diff
@@ -567,7 +567,7 @@ func (f *Fs) About(ctx context.Context) (*fs.Usage, error) {
 // This should return ErrDirNotFound if the directory isn't
 // found.
 func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err error) {
-	entriess := make([][]upstream.Entry, len(f.upstreams))
+	entriesList := make([][]upstream.Entry, len(f.upstreams))
```
Comment (jsoref) [inline backend/union/union.go:570]: I'm happy to drop this (and a similar change) if requested.

It's a lot easier to understand a code block if it does not have two variables in a scope that differ by only a single letter.

File: backend/webdav/webdav_test.go [hunk 7]
```diff
@@ -12,7 +12,7 @@ import (
 // TestIntegration runs integration tests against the remote
 func TestIntegration(t *testing.T) {
 	fstests.Run(t, &fstests.Opt{
-		RemoteName: "TestWebdavNexcloud:",
+		RemoteName: "TestWebdavNextcloud:",
```
Comment (jsoref) [inline backend/webdav/webdav_test.go:15]: I presume this isn't a big deal

File: cmd/info/info.go [hunk 8]
```diff
@@ -49,7 +49,7 @@ func init() {
 	flags.BoolVarP(cmdFlags, &checkControl, "check-control", "", true, "Check control characters.")
 	flags.DurationVarP(cmdFlags, &uploadWait, "upload-wait", "", 0, "Wait after writing a file.")
 	flags.BoolVarP(cmdFlags, &checkLength, "check-length", "", true, "Check max filename length.")
-	flags.BoolVarP(cmdFlags, &checkStreaming, "check-streaming", "", true, "Check uploadxs with indeterminate file size.")
+	flags.BoolVarP(cmdFlags, &checkStreaming, "check-streaming", "", true, "Check uploads with indeterminate file size.")
```
Comment (jsoref) [inline cmd/info/info.go:52]: I presume this isn't some special term...

File: fs/accounting/transfermap.go [hunk 9]
```diff
@@ -91,7 +91,7 @@ func (tm *transferMap) _sortedSlice() []*Transfer {
 func (tm *transferMap) String(progress *inProgress, exclude *transferMap) string {
 	tm.mu.RLock()
 	defer tm.mu.RUnlock()
-	strngs := make([]string, 0, len(tm.items))
+	stringList := make([]string, 0, len(tm.items))
```
Comment (jsoref) [inline fs/accounting/transfermap.go:94]: This is that second instance...

File: fstest/testserver/init.d/TestWebdavOwncloud [hunk 10]
```diff
@@ -12,7 +12,7 @@ start() {
     docker run --rm -d --name $NAME \
            -e "OWNCLOUD_DOMAIN=${OWNCLOUD_DOMAIN}" \
            -e "OWNCLOUD_DB_TYPE=sqlite" \
-           -e "OWNCLOUD_DB_NAME=oowncloud.db" \
+           -e "OWNCLOUD_DB_NAME=owncloud.db" \
```
Comment (jsoref) [inline fstest/testserver/init.d/TestWebdavOwncloud:15]: I presume this is ok. it's possible it's intentional, in which case it'd be helpful to understand why...

File: lib/structs/structs_test.go [hunk 11]
```diff
@@ -57,7 +57,7 @@ func TestSetFrom(t *testing.T) {
 		Matching:      "a",
 		OnlyA:         "onlyA",
 		MatchingInt:   1,
-		DifferentType: "suprise",
+		DifferentType: "surprise",
```
Comment (jsoref) [inline lib/structs/structs_test.go:60]: ?

**Comment by jsoref** [comment]
The mac build failure is odd. But it's long before any code runs...

File: MANUAL.html (hunk 2 above)
Comment (ncw) [reply MANUAL.html:6568]: LGTM

File: backend/jottacloud/api/types.go (hunk 3 above)
Comment (ncw) [reply backend/jottacloud/api/types.go:158]: I don't think they are.

File: backend/jottacloud/jottacloud.go (hunk 4 above)
Comment (ncw) [reply backend/jottacloud/jottacloud.go:110]: :-) LGTM

File: backend/onedrive/api/types.go (hunk 5 above)
Comment (ncw) [reply backend/onedrive/api/types.go:257]: LGTM

File: backend/union/union.go (hunk 6 above)
Comment (ncw) [reply backend/union/union.go:570]: I think `entriess` is taking it a bit far... so this change is fine!

File: backend/webdav/webdav_test.go (hunk 7 above)
Comment (ncw) [reply backend/webdav/webdav_test.go:15]: Looks like you found a bug there! This needs to match the name in `fstest/testserver/init.d/TestWebdavNextcloud`

File: cmd/info/info.go (hunk 8 above)
Comment (ncw) [reply cmd/info/info.go:52]: he he

File: fstest/testserver/init.d/TestWebdavOwncloud (hunk 10 above)
Comment (ncw) [reply fstest/testserver/init.d/TestWebdavOwncloud:15]: I think it is an accident, so your fix is good

File: lib/structs/structs_test.go (hunk 11 above)
Comment (ncw) [reply lib/structs/structs_test.go:60]: That is fine

**Comment by ncw** [comment]
Thanks for doing this - very nice :-)

There are some auto generated files in there

```
MANUAL.*
rclone.1
docs/content/commands/*.md
```

I can see it might be a pain to fix that... So what I could do is squash the commits (which I'll probably do anyway) and drop the autogenerated files from the combined commit. Sound OK? Or do you want do it?

> The mac build failure is odd. But it's long before any code runs...

Yes it is a Github actions failure I think.

**Comment by jsoref** [comment]
I'm quite happy to let you do all of it - Thanks. (I'm not attached to the work.)

Sometimes it's vaguely easy for me to drop things, I think I could do it here (it's a couple of passes: check out a base, delete the files, commit, rebase onto, drop conflicts, rebase back to original base).

File: backend/webdav/webdav_test.go (hunk 7 above)
Comment (jsoref) [reply backend/webdav/webdav_test.go:15]: Yeah, that's not surprising.

Often fixes like this end up showing that tests don't test what they think they're testing :-o.

If 91 commits only found one bug, that's actually pretty good :-)

**Comment by jsoref** [comment]
@ncw: seems ok, is this just a flaky test?

**Comment by jsoref** [comment]
@ncw: I've updated my test and it likes this branch. I presume that the windows_386 build failure is a flaky test?

**Comment by ncw** [comment]
> @ncw: I've updated my test and it likes this branch. I presume that the windows_386 build failure is a flaky test?

Yes that is a flaky test.

I'll merge this now :-)

Thank you very much :-)

**Pull request merged**